tests/pyrog/api/test_source.py

```diff
@@ -90,28 +90,6 @@ def test_can_import_mapping_several_times(api_client, export_data):
     assert len(response.data) == 2
 
 
-@pytest.mark.as_user
-def test_retrieve_full_source(
```

Comment:
Why has this been removed?

Reply:
the query param full=true doesn't exist anymore